### ISSUE TITLE
Mass emails created using a Quick Reply template and CSV import are not reporting tracking information correctly. Instead of the name of the recipient, tracking just reports that "Someone" opened the email.  Have to click the link to see who opened it, which is not particularly convenient for mass emails.

### ISSUE THREAD

**Issue by meritweb** [nylas/N1#2982]
##### Are there any related issues?

...

##### What operating system are you using?
...

##### What version of N1 are you using?
...

--

**Bug?**
##### Do you have any third-party plugins installed? If so, which ones?
...

##### Is the issue related to a specific email provider (Gmail, Exchange, etc.)?
...

##### Is the issue reproducible with a particular attachment, message, signature, etc?

...

--

**Feature Request?**
##### Does this feature exist in another mail client or tool you use?
...

**Comment by jackiehluo** [comment]
@meritweb Which email provider are you using? And are you sending the mail merge emails to one person per email?

**Comment by meritweb** [comment]
Gmail (specifically, Apps for Education), and yes, the emails are going to one person per email.  It's happened with 2 batches now, each between 40 and 50 emails to individual email addresses.

**Comment by bengotow** [comment]
@jackiehluo  I think this is actually a known issue - we don't enable per-recipient read receipts with Mail Merge. That said, if the email is only sent to one recipient at a time, we should still know to show their name rather than Someone... will investigate.

**Comment by jackiehluo** [comment]
Yeah, exactly—for any email that's sent with only one recipient, we should just default to the recipient's name (I thought we already did that, but I'm not sure!).

**Comment by meritweb** [comment]
If the name is pulled from Google Contacts, that might be the issue here - these batches of emails were imported directly from a CSV and were not in Contacts. If this is the issue, would it be possible to fall back to an email address if no name is present? That would avoid the Someone issue.
